parsers/CA_NB.py

```diff
@@ -74,25 +88,29 @@ def fetch_production(
         + flows["PEI"]
         + flows["QUEBEC"]
     )
+    production = ProductionBreakdownList(logger)
+    production.append(
+        zoneKey=zone_key,
+        # Using the current utc time because the page returns the current time.
+        datetime=datetime.now(tz=timezone.utc).replace(
+            minute=0, second=0, microsecond=0
+        ),
+        source=SOURCE,
+        production=ProductionMix(
+            unknown=generated,
+        ),
```

Comment:
not part of this PR but would it make sense to switch these to use totalgeneration instead at some point instead of adding it as unknown to the production mix? (Would probably require other changes.)

Reply:
Yeah I was also thinking that. The problem is that currently the load data is not taken into account and therefore not shown on the map. But I'm thinking of creating an estimation model that would take the load and put it into unknown, seems like the best thing to do for me.

Reply:
Yeah, that way we don't pollute the database either.